docker/iroh-fedimintd/docker-compose.yaml

```diff
@@ -10,8 +12,22 @@ services:
       - FM_BITCOIN_NETWORK=signet
       - FM_BITCOIN_RPC_KIND=esplora
       - FM_BITCOIN_RPC_URL=https://mutinynet.com/api
-      - FM_BIND_UI=0.0.0.0:8175
+      - FM_BIND_P2P=0.0.0.0:8173
+      - FM_BIND_API_IROH=0.0.0.0:8174
+      - FM_BIND_API_WS=172.25.0.11:8174
+      - FM_BIND_UI=172.25.0.11:8175
+    networks:
+      - fedimint_network
+        ipv4_address: 172.25.0.11
```

Comment:
This is invalid

```
yaml: line 21: mapping values are not allowed in this context
```

We need to define as

```
    networks:
      fedimint_network:
        ipv4_address: 172.25.0.11
```

Did this work for you locally? If so, it's likely the same reason why the duplicate `restart` policies worked on my old vm since it was using the old unsupported docker version

Reply:
I copied it from somewhere.